src/utils/watcher.go

```diff
@@ -3,9 +3,10 @@ package utils
 import (
 	"bytes"
 	"encoding/json"
+	"io"
 	"io/fs"
```

Comment:
Any point importing io and io/fs? Or is it not like JavaScript?

Reply:
You have to do it this way afaik, `fs` is an extended namespace of `io`

Reply:
Oh wait nvm, you can get the same type from `os`